internal/core/data/router_test.go

```diff
@@ -67,10 +68,7 @@ var TestValueDescriptor3 = contract.ValueDescriptor{
 }
 
 func TestRestValueDescriptorsUsageHandler(t *testing.T) {
-	Configuration = &ConfigurationStruct{Service: config.ServiceInfo{MaxResultCount: 10}}
 	defer func() { Configuration = &ConfigurationStruct{} }()
```

Comment:
(You're going to hate me.) 

Clean up should occur at the same level of allocation/change.  

Move this down below the `Configuration` assignment in the `t.Run()` method.

Reply:
Hahaha, I never can hate someone taking the time to make MY code better. I appreciate it. Plus you are right :)

Reply:
Actually this is my fault. I should have looked closer. My apologises.